.github/policies/resourceManagement.yml

```diff
@@ -233,5 +233,14 @@ configuration:
       - addReply:
           reply: Hi! Thanks for making us aware of the problem. We raised the issue with our internal localization team. This issue should be fixed hopefully in the next version of PowerToys.
       description: 
+    - if:
+      - payloadType: Issue_Comment
+      - commentContains:
+          pattern: 'like to help|like helping|would like to contribute'

```

Comment:
and `... fix this`, `... implement this`

Maybe we should required `I ... like ...` to prevent false positives.